website/source/docs/providers/postgresql/r/postgresql_database.html.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: "postgresql"
+page_title: "PostgreSQL: postgresql_database"
+sidebar_current: "docs-postgresql-resource-postgresql_database"
+description: |-
+  Creates and manages a database on a PostgreSQL server.
+---
+
+# postgresql\postgresql_database
```

Comment:
I think this should be:

```markdown
# postgresql\_database
```

The backslashes we have on these are to ensure that multiple underscores don't get misinterpreted by the Markdown parser as an underline directive. I expect it won't actually be a problem in this case, since one underscore is not enough to cause an underline, but having it there is consistent with how other resources are documented.